### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by cakuros**
Included doc changes
- In the FAQ, the space in the name of the mapping is technically a naming violation, even if it is intended to be a placeholder
- In the Helm Install, making it more clear that the edgectl install portion of the document is optional, and that
- Scrubbed Edge Stack from the basic auth and basic rate limiting howtos, since they are specific to API Gateway only, also made a note at the top reinforcing that these howtos are not supported by AES.
  - Also pulled some yaml annotations out of the service definition in the RL howto.

This PR only addresses v1.6 documentation, if PR is accepted I will create more for 1.3-1.5.

### CODE REVIEW

File: docs/howtos/basic-auth.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
 # Authentication
 
-Ambassador can authenticate incoming requests before routing them to a backing service. In this tutorial, we'll configure Ambassador to use an external third party authentication service. Note that if you're using the Ambassador Edge Stack, the [`External` filter](../../topics/using/filters) is a more powerful way to manage authentication. 
+IMPORTANT: This guide applies to the Ambassador API Gateway version of Ambassador, use of this guide on the Ambassador Edge Stack is not fully supported; use the [External Filter](../../topics/using/filters) instead.
+
+Ambassador can authenticate incoming requests before routing them to a backing service. In this tutorial, we'll configure Ambassador to use an external third party authentication service. Note that if you're using the Ambassador Edge Stack, the [`External` filter](../../topics/using/filters) is the supported way to manage authentication.
```
Comment (khussey) [inline docs/howtos/basic-auth.md:5]: The second sentence here seems redundant now that you've already stated above to use the external filter for AES...

File: docs/howtos/basic-auth.md [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
 
 - listens for requests on port 3000;
 - expects all URLs to begin with `/extauth/`;
 - performs HTTP Basic Auth for all URLs starting with `/backend/get-quote/` (other URLs are always permitted);
 - accepts only user `username`, password `password`; and
 - makes sure that the `x-qotm-session` header is present, generating a new one if needed.
 
-Ambassador Edge Stack routes _all_ requests through the authentication service: it relies on the auth service to distinguish between requests that need authentication and those that do not. If Ambassador Edge Stack cannot contact the auth service, it will return a 503 for the request; as such, **it is very important to have the auth service running before configuring Ambassador Edge Stack to use it.**
+Ambassador routes _all_ requests through the authentication service: it relies on the auth service to distinguish between requests that need authentication and those that do not. If Ambassador cannot contact the auth service, it will return a 503 for the request; as such, **it is very important to have the auth service running before configuring Ambassador Edge Stack to use it.**
```
Comment (khussey) [inline docs/howtos/basic-auth.md:23]: "Ambassador Edge Stack" -> "Ambassador" at the end of this paragraph?

File: docs/howtos/rate-limiting-tutorial.md [hunk 3]
```diff
@@ -72,11 +74,11 @@ spec:
 
 This configuration tells the Ambassador Edge Stack about the rate limit service, notably that it is serving requests at `example-rate-limit:5000`.
 
-The Ambassador Edge Stack will see the RateLimitService and reconfigure itself within a few seconds. Note that the v2 API is available for the Ambassador Edge Stack.
+The Ambassador will see the RateLimitService and reconfigure itself within a few seconds. Note that the v2 API is available for the Ambassador.
```
Comment (khussey) [inline docs/howtos/rate-limiting-tutorial.md:77]: "The Ambassador" -> "Ambassador".

File: docs/howtos/rate-limiting-tutorial.md [hunk 4]
```diff
@@ -1,10 +1,12 @@
 # Rate Limiting
 
-The Ambassador Edge Stack can validate incoming requests before routing them to a backing service. In this tutorial, we'll configure the Ambassador Edge Stack to use a simple third party rate limit service. If you don't want to implement your own rate limiting service, the Ambassador Edge Stack integrates a [powerful, flexible rate limiting service](../../topics/using/rate-limits/).
+IMPORTANT: This guide applies to the Ambassador API Gateway version of Ambassador, use of this guide on the Ambassador Edge Stack is not fully supported; use the existing [RateLimitService](../../topics/using/rate-limits/) instead.
```
Comment (khussey) [inline docs/howtos/rate-limiting-tutorial.md:3]: "version of Ambassador, use" -> ". Use"

File: docs/howtos/basic-auth.md (hunk 1 above)
Comment (cakuros) [reply docs/howtos/basic-auth.md]: Fixed

File: docs/howtos/basic-auth.md (hunk 2 above)
Comment (cakuros) [reply docs/howtos/basic-auth.md]: Fixed

File: docs/howtos/rate-limiting-tutorial.md (hunk 3 above)
Comment (cakuros) [reply docs/howtos/rate-limiting-tutorial.md]: Fixed

File: docs/howtos/rate-limiting-tutorial.md (hunk 4 above)
Comment (cakuros) [reply docs/howtos/rate-limiting-tutorial.md]: Fixed

**Comment by cakuros** [comment]
Yes, I tested the mappings separately.  Also found a handful of references to Edge Stack in the rate limiting tutorial that I had missed previously.

File: docs/howtos/basic-auth.md [hunk 5]
```diff
@@ -1,24 +1,26 @@
 # Authentication
 
-Ambassador can authenticate incoming requests before routing them to a backing service. In this tutorial, we'll configure Ambassador to use an external third party authentication service. Note that if you're using the Ambassador Edge Stack, the [`External` filter](../../topics/using/filters) is a more powerful way to manage authentication. 
+IMPORTANT: This guide applies to the Ambassador API Gateway version of Ambassador, use of this guide on the Ambassador Edge Stack is not fully supported; use the [External Filter](../../topics/using/filters) instead.
```
Comment (khussey) [inline docs/howtos/basic-auth.md:3]: "Ambassador API Gateway version of Ambassador" reads awkwardly. Instead, I would say this:

IMPORTANT: This guide applies to the Ambassador API Gateway. Use of this guide with the Ambassador Edge Stack is not fully supported; use ...

File: docs/howtos/rate-limiting-tutorial.md [hunk 6]
```diff
@@ -1,16 +1,18 @@
 # Rate Limiting
 
-The Ambassador Edge Stack can validate incoming requests before routing them to a backing service. In this tutorial, we'll configure the Ambassador Edge Stack to use a simple third party rate limit service. If you don't want to implement your own rate limiting service, the Ambassador Edge Stack integrates a [powerful, flexible rate limiting service](../../topics/using/rate-limits/).
+IMPORTANT: This guide applies to the Ambassador API Gateway version of Ambassador, use of this guide on the Ambassador Edge Stack is not fully supported.  Use the existing [RateLimitService](../../topics/using/rate-limits/) instead.
```
Comment (khussey) [inline docs/howtos/rate-limiting-tutorial.md:3]: I would use the same phrasing here as (what I suggested) for the basic auth guide, i.e., something more like:

IMPORTANT: This guide applies to the Ambassador API Gateway. Use of this guide with the Ambassador Edge Stack is not fully supported; use the more powerful, flexible ...

Comment (cakuros) [reply docs/howtos/rate-limiting-tutorial.md]: Fixed

File: docs/howtos/basic-auth.md (hunk 5 above)
Comment (cakuros) [reply docs/howtos/basic-auth.md]: Fixed

**Pull request merged**